docs/essentials/collections/collections-and-data-structures.md

```diff
@@ -55,13 +55,13 @@ Manage efficient storage and access of unique elements | [System.Collections.Gen
-[When to Use Generic Collections](whentoUseGenericCollections.md) | Discusses the use of generic collection types.
-[Comparisons and Sorts Within Collections](comparisonsandSortsWithinCollections.md) | Discusses the use of equality comparisons and sorting comparisons in collections.
-[Sorted Collection Types](sortedCollectionTypes.md) | Describes sorted collections performance and characteristics.
-[Hashtable and Dictionary Collection Types](hashtableandDictionaryCollectionTypes.md) | Describes the features of generic and non-generic hash-based dictionary types.
-[Thread-Safe Collections](thread-safeCollections.md) | Describes collection types such as [System.Collections.Concurrent.BlockingCollection&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.BlockingCollection%601.html) and [System.Collections.Concurrent.ConcurrentBag&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.ConcurrentBag%601.html) that support safe and efficient concurrent access from multiple threads.
+[Selecting a Collection Class](selecting-a-collection-class.md) | Describes the different collections and helps you select one for your scenario.
+[Commonly Used Collection Types](commonly-used-collection-types.md) | Describes commonly used generic and nongeneric collection types such as [System.Array](http://dotnet.github.io/api/System.Array.html), [System.Collections.Generic.List&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.List%601.html), and [System.Collections.Generic.Dictionary&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Generic.Dictionary%602.html). 
+[When to Use Generic Collections](when-to-use-generic-collections.md) | Discusses the use of generic collection types.
+[Comparisons and Sorts Within Collections](comparisons-and-sorts-within-collections.md) | Discusses the use of equality comparisons and sorting comparisons in collections.
+[Sorted Collection Types](sorted-collection-types.md) | Describes sorted collections performance and characteristics.
+[Hashtable and Dictionary Collection Types](hashtable-and-dictionary-collection-types.md) | Describes the features of generic and non-generic hash-based dictionary types.
+[Thread-Safe Collections](thread-safe-collections.md) | Describes collection types such as [System.Collections.Concurrent.BlockingCollection&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.BlockingCollection%601.html) and [System.Collections.Concurrent.ConcurrentBag&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.ConcurrentBag%601.html) that support safe and efficient concurrent access from multiple threads.
```

Comment:
@stevehoag Yeah, I fixed all the links inside the docs. docfx, does not report any link errors.